lib/broccoli/ember-app.js

```diff
@@ -1534,25 +1536,29 @@ EmberApp.prototype.toTree = function(additionalTrees) {
  */
 EmberApp.prototype.contentFor = function(config, match, type) {
   var content = [];
-  var deprecatedHooks = ['app-prefix', 'app-suffix'];
+  var privateHooks = ['app-config'];
```

Comment:
lets extract these to the top of the file as constants, as we do not change them.